activemq-broker/src/main/java/org/apache/activemq/broker/BrokerService.java

```diff
@@ -485,15 +490,15 @@ public JmsConnector removeJmsConnector(JmsConnector connector) {
     }
 
     public void masterFailed() {
-        if (shutdownOnMasterFailure) {
-            LOG.error("The Master has failed ... shutting down");
+        if (shutdownOnActiveFailure) {
+            LOG.error("The Active has failed ... shutting down");
```

Comment:
Active doesnt sound like a good replacement master
 Typically we would refer to master or slave being the current active node. 

Active = state

Master / slave was what role the node took. 

I think a different word to replace master should be found

Reply:
The [ticket](https://issues.apache.org/jira/browse/AMQ-7514) mentions active/standby and I'd tend to agree that it's the clearest (it's also what we use at AWS).

Consider:
- of the nodes, only 1 is _active_ serving requests
- the other nodes are not pure replicas, as they have shared storage, rather they are on standby over the same FS

It's possible that in the previous replicated leveldb one could think of replica as a good term here, but I think it's slightly disingenuous. (especially if the storage used isn't actually replicated...)

Reply:
I disagree with the ticket then
Active is state though. Either master or slave can be active. We will end up with very confusing terminology.

https://www.theserverside.com/opinion/Master-slave-terminology-alternatives-you-can-use-right-now

Reply:
Quickly reading the ticket actually mentions

New terms:
a. For shared storage: ‘active’ and ’standby’
b. For replication: ‘primary’ and ‘replica'
c. For 'white list' and 'blacklist': 'allow list' and 'deny list'

This is down to replication state so primary replica should be used

Reply:
I agree with @michaelandrepearce. I've added a comment to AMQ-7514 to push along the discussion.

Reply:
This group of methods is never read anywhere by the code base. My understanding is that they are legacy hold-over from early replication approach that has been removed long ago. I do not see any reason to deprecate them. Given that we have agreed to make major breaking changes in 5.17.x (remove leveldb), I believe we can safely remove these methods in this pass.

We should not be adding new terms or concepts that do not apply to the code base in this pass.

Reply:
I did a 'find references' search and yeah, you're right @mattrpav. I think this is also relevant in the discussion about active/passive/standby. It seems reasonable in the scope of this change rather than renaming to actually remove those methods/concepts. Then there is no need to consider the 'existing' terminology.

Reply:
as per @mattrpav 's AMQ-8316, moved the two concretely deprecated messages out of this PR